lib/l10n/intl_it.arb

```
@@ -831,6 +831,19 @@
   "add": "Aggiungi",
 
   "save": "Salva",
+  "apply": "Applica",
+
+  "@_comment_filter_section": "Stringhe della sezione filtri",
+  "fiatCurrencies": "Valute fiat",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)